static/styles/portico.scss

```diff
@@ -371,6 +403,10 @@ li div.codehilite {
     margin-bottom: 60px;
 }
 
+#page-title {
```

Comment:
You should probably prefix `digest` in all the classes and ids.

Reply:
@hackerkid  I've made the changes. Could you please check?